### DISCUSSION TITLE
How to enable https

### DISCUSSION THREAD

**Discussion by qunshen** [oqtane/oqtane.framework#1889] [Q&A] [answered]
how to enable https in Oqtane?

**Comment by hishamco** [comment]
You can add normal Https like any other ASP.NET Core app in startup. BTW you can create your custom startup and add it if not exists by default

**Comment by sbwalker** [comment]
@qunshen Startup already contains app.UseHttpsRedirection(); so I am not sure what you are asking?

**Comment by qunshen** [comment]
what I want is that When I launch Oqtane from Visual Studio, the site is https://localhost:44357/. now When I lauch Oqtane, the url is http://localhost:44357/

**Reply by leigh-pointer** [reply]
Look at this https://docs.microsoft.com/en-us/iis/manage/configuring-security/how-to-set-up-ssl-on-iis

**Comment by qunshen** [comment]
my question is that how to set https at local, during development, without a commercial certificate

**Reply by leigh-pointer** [reply]
https://dontpaniclabs.com/blog/post/2017/01/11/creating-self-signed-ssl-certificate-developing-testing-iis/amp/

**Comment by qunshen** [comment, marked as answer]
Thanks

**Comment by hishamco** [comment]
Strange that's no answer was proper for your question and you marked yours ;)
